app/controllers/Application.java

```diff
@@ -607,13 +616,8 @@ private static Result getFlowHistory(Version version) {
         return ok(flowHistoryPage.render(flowDefPair.getId(), graphType,
             flowHistoryResults.render(flowDefPair, executionMap, idPairToJobNameMap, flowExecTimeList)));
       } else if (graphType.equals("resources") || graphType.equals("time")) {
-        if (hasSparkJob) {
```

Comment:
We're now computing aggregated metrics for spark as well hence removed this check. This check was only to have history pages for non spark jobs, now we can have them. I haven't removed the above code since it still applies to the older job history and flow history graphs. Once we deprecate the older UI, I'll clean it up and remove it.